packages/bbui/vite.config.js

```diff
@@ -0,0 +1,29 @@
+import { defineConfig } from "vite"
+import { svelte } from "@sveltejs/vite-plugin-svelte"
+import postcss from "postcss"
+import path from "path"
+
+export default defineConfig({
+  build: {
+    sourcemap: true,
+    lib: {
+      entry: "src/index.js",
+      formats: ["es"],
+      fileName: "bbui.es",
+    },
+  },
+  plugins: [
+    svelte({
+      emitCss: true,
+    }),
+  ],
+  css: {
+    postcss,
+  },
```

Comment:
I don't think we need to include the default postcss plugin - I think vite should handle it natively.

Reply:
I saw that not having this created a different output, but it all works fine without it!